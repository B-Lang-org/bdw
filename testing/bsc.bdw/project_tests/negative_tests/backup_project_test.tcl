
# Backup the project
handlers::backup_project project.tgz -input_files

# Backup the project
handlers::backup_project

# Opens the specified project
handlers::open_project project.bspec

# Backup the project
handlers::backup_project project.tgz -input_fis

# Backup the project
handlers::backup_project project.tgz -project_d

# Backup the project
handlers::backup_project project.tgz -search_pat

# Backup the project
handlers::backup_project project.tgz -input_fies -poject_dir

# Backup the project
handlers::backup_project

# Backup the project
handlers::backup_project project.tgz

# Backup the project
handlers::backup_project project.tgz -options -v

# Backup the project
handlers::backup_project -options -v

# Backup the project
handlers::backup_project project.tgz -input_files -project_dir -search_path -options

# Backup the project
handlers::backup_project project.tgz -input_files -project_dir -search_path -options -v -search_path_fles

# Backup the project
handlers::backup_project project.tgz -input_files -project_dir -search_path -options -v -search_path_fles

# Closes the project
handlers::close_project

# Exits the workstation
exit
