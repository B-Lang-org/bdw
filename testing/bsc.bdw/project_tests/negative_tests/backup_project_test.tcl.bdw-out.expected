There is no open project
There is no open project
Unknown option: -input_fis
Unknown option: -project_d
Unknown option: -search_pat
Unknown option: -input_fies
Archive file name should be specified.
Files to be backed up should be specified.
Files to be backed up should be specified.
Unknown option: -v
Argument -options expects an argument none found
Unknown option: -search_path_fles
Unknown option: -search_path_fles
